### PR TITLE
New version of siliconvalleyinsight.github.io (Beta)

### PR DESCRIPTION
**Pull request by alexnodejs**
@JohnYangSam Please check.

List of **unimplemented** features:
- Advanced scrolling
- Fonts (I use Open Sans for now)

Since this is an urgent task, I will try to create additional PR(s) on this weekend which should include "advanced scrolling features".

P.S. please see this who don't want to test.
http://screencast.com/t/iPYdNGCak

### CODE REVIEW

**Comment by JohnYangSam** [comment]
@alexnodejs added copy. A few pieces of feedback:
- Can we center the location images better (underneath the text)
- When the page is wide enough we should see the `About` header align with `What we do` and `Our Experience`
- The copy changes also changed alighment of those latter two sections, they should be aligned from the top instead of the bottom
- When the page is shrunk to mobile, I think we need more white space below the main header
- Check the chrome console output - I was getting a few JS errors
- Was getting a few odd characters in Sublime. What editor are you using?
  screen shot 2015-07-18 at 12 39 56 am
- Attaching the [designs here](http://cl.ly/361v3w3p1t0a) for reference

File: index.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: index
 ---
+
 <script>
   (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
```
Comment (benigeri) [inline index.html]: is this scrypt async? If not probs better to put it below the content.

**Comment by JohnYangSam** [comment]
You've got the goahead to merge this into the main branch to display on the website branch once the tweaksa re updated!

**Comment by alexnodejs** [comment]
@JohnYangSam 
- Most of the feedbacks related to the alignment and of course I will align it better when I work with scroll.
- I don't see any errors in Chrome http://screencast.com/t/UPO8UbNTzPL7, please provide what you see.
- _Was getting a few odd characters in Sublime. What editor are you using?_ - I use web storm and yes I'll remove them it was happened when I copy pate from photoshop.
- I saw design and I have this file, it was a basic version without alignment which should be added when I'll implement "scrolling features" which I mentioned above.

File: index.html (hunk 1 above)
Comment (alexnodejs) [inline index.html]: @benigeri Let's move it there

**Comment by alexnodejs** [comment]
@JohnYangSam I'm going to merge this PR with master, it seems all is aligned now.

Please see.
http://screencast.com/t/kr8i9z5MZi8

Also please give me some review / feedback.

**Comment by JohnYangSam** [comment]
@alexnodejs good updates.

@benigeri mentioned that since we are hijacking the scrolling, it might be causing the weird scrolling on the right (and the extra bounce on both the left and the right when we get to the bottom). For now, on the main branch, could we see what it looks like without the scrolling?

Also, I think I see what you were trying to do with the scrolling at the end to align with the countries. I don't think we need that for now - the smooth scrolling might be better for the moment.

Let's sync with denis on designs.

**Comment by alexnodejs** [comment]
@JohnYangSam, @benigeri 

I will provide version without "scroll to fixed" functionality
in a PR from next branch soon https://github.com/SiliconValleyInsight/siliconvalleyinsight.github.io/tree/alexv/fixes-for-new-site-version

**Pull request merged**